web/styles/left_sidebar.css

```diff
@@ -403,6 +405,67 @@
     }
 }
 
+.stream-list-toggle-inactive-channels {
+    padding-left: var(--left-sidebar-toggle-width-offset);
+
+    .show-inactive-channels,
+    .hide-inactive-channels {
+        /* Override the action heading font size, so that em measurements
```

Comment:
This is for a follow-up, but I wonder whether the "action heading" font size should be inheriting to avoid this problem. I'm not entirely sure which font size you're talking about here.